src/usdb_syncer/usdb_scraper.py

```diff
@@ -525,3 +525,32 @@ def _parse_song_txt_from_txt_page(soup: BeautifulSoup) -> str:
     if isinstance(textarea := soup.find("textarea"), Tag):
         return textarea.string or ""
     raise errors.UsdbParseError("textarea for notes not found")
+
+
+def post_song_comment(song_id: SongId, text: str, rating: str, logger: Log) -> None:
+    """Post a song comment to USDB."""
+    payload = {"text": text, "stars": rating}
+
+    get_usdb_page(
+        "index.php",
+        RequestMethod.POST,
+        headers={"Content-Type": "application/x-www-form-urlencoded"},
+        params={"link": "detail", "id": str(int(song_id)), "comment": str(1)},
+        payload=payload,
+    )
+    logger.debug(f"#{song_id}: Comment posted on USDB.")
```

Comment:
`logger` might be a `SongLogger` which already prepends the song id.

Reply:
But it isn't. I get `2025-03-09 17:48:27 [DEBUG] #30096: Comment posted on USDB.` How might it be a `SongLogger`?

Reply:
It's about the type. `Log` might be a `SongLogger`. If it has to be a `SongLogger`, then the type annotation should say so. However, since you have the song id, you don't need to passed the logger at all. Just get it from the logger module.

Reply:
Ah, got it!